docs/relay-deployment.md

```diff
@@ -0,0 +1,148 @@
+# Manual setup for a GSN relayer
+
+## Compile the server
+
+On Linux:
+```
+yarn 
+make server
+cp build/server/bin/RelayHttpServer .
+```
+
+On Mac (using docker to build the linux image):
```

Comment:
Change "on mac" to "with docker", and add prerequisites on linux (Go, truffle etc.)